pkgs/applications/misc/netsurf/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv, fetchurl, SDL, pkgconfig, libpng, openssl, curl, gtk2, check
+, libxml2, libidn, perl, nettools, spidermonkey_185, nspr, perlPackages
+, netsurf-buildsystem
+, netsurf-nsgenbind
+, netsurf-libnsfb
+, netsurf-libwapcaplet
+, netsurf-libparserutils
+, netsurf-libcss
+, netsurf-libhubbub
+, netsurf-libdom
+, netsurf-libnsbmp
+, netsurf-libnsgif
+, netsurf-libnsutils
+, netsurf-libutf8proc
+}:
+
+stdenv.mkDerivation rec {
+
+  name = "netsurf-${version}";
+  version = "3.5";
+
+  # UIS incldue Framebuffer, and gtk, but
```

Comment:
It isn't exposed but there is some branching on it. If it is essentially dead code, my preference would be to remove it, but if you intend to work on framebuffer support, then feel free to leave it in. Note that you are not obligated to support every feature the package may expose, so unless you have a personal interest in the framebuffer support, I think a gtk only offrering is completely okay.